src/fmripost_aroma/workflows/base.py

```diff
@@ -207,6 +207,15 @@ def init_single_subject_wf(subject_id: str):
         # Patch standard-space BOLD files into 'bold' key
         subject_data['bold'] = listify(subject_data['bold_mni152nlin6asym'])
 
+    if not subject_data['bold_mni152nlin6asym']:
```

Comment:
It looks like this was a mistake for raw+derivatives processing, because the derivatives datasets weren't being queried.